src/main/java/com/pubnub/api/workers/SubscribeMessageProcessor.java

```diff
@@ -166,22 +168,10 @@ private BasePubSubResult getBasePubSubResult(SubscribeMessage message, String ch
     }
 
     private boolean isMessage(SubscribeMessage message) {
```

Comment:
In this case I would keep comment here because IntelliJ informing that  "==" is not proper way of doing comparison(Strings and Boxed types should be compared using "equals()".) and provide quick fix that once applied will cause method to work improperly. 

Reg. second comparison. I think is correct. The thing is that message.getPnMessageType() can be null. When is null then it means that subscribeMessage is of time "message" not "signal" or "file" of "messageActon". The same applies when message.getPnMessageType() = 0 indicating this it "message".